### DISCUSSION TITLE
Lerna publish rate limited exceeded ?

### DISCUSSION THREAD

**Discussion by wangchaodeyuzhou** [lerna/lerna#4023] [Q&A]
![image](https://github.com/lerna/lerna/assets/48428618/93aa3e1d-c534-4191-9c4b-ea760915c458)
lerna ERR! E429 Could not publish, as user undefined: rate limited exceeded

**Comment by ghiscoding** [comment]
Perhaps the newly merged and released PR #4013 can help with this?
